sdk/edgeupdates/Azure.ResourceManager.EdgeUpdates/src/autorest.md

```diff
@@ -0,0 +1,49 @@
+# Generated code configuration
+
+Run `dotnet build /t:GenerateCode` to generate code.
+
+``` yaml
+azure-arm: true
+csharp: true
+library-name: Edge
+namespace: Azure.ResourceManager.Edge
+require: https://raw.githubusercontent.com/Azure/azure-rest-api-specs-pr/bad587f7237e08e93cd45e95988f8a95b30c23c1/specification/edge/updates/resource-manager/readme.md?token=GHSAT0AAAAAACCFCF22OBZDHAHA24F5YGAGZFXXEFQ
```

Comment:
This repo is for public SDK generation, can't point to private spec repo.

Reply:
The token has been removed, thanks ! The swagger PR for this sdk is still in review, I will update the public branch as soon as the PR is merged.